backend/cmd/headlamp_test.go

```diff
@@ -649,6 +649,58 @@ func TestHandleClusterAPI_XForwardedHost(t *testing.T) {
 	assert.Equal(t, "OK", rr.Body.String())
 }
 
+// Handle a cluster rename request.
```

Comment:
Please follow golang style comments, and follow the style of the other comments in the file? Start with the function name.

Reply:
thanks